Xamarin.Forms.Platform.UAP/ImageButtonRenderer.cs

```diff
@@ -71,7 +71,11 @@ protected async override void OnElementChanged(ElementChangedEventArgs<ImageButt
 			{
 				if (Control == null)
 				{
-					_formsButton = new FormsButton();
+					_formsButton = new FormsButton
+					{
+						Style = Windows.UI.Xaml.Application.Current.Resources["FormsButtonStyle"] as Windows.UI.Xaml.Style
+					};
+
 					_formsButton.Padding = new WThickness(0);
```

Comment:
Shouldn't these properties (Padding, BorderThickness and Background) be set in the Style now?